.github/libs/GithubUtils.ts

```diff
@@ -577,11 +577,10 @@ class GithubUtils {
         console.log('Getting pull requests merged between the following tags:', fromTag, toTag);
 
         try {
-            const {data: comparison} = await this.octokit.repos.compareCommits({
+            const comparison = await this.paginate(this.octokit.repos.compareCommitsWithBasehead, {
```

Comment:
Using `compareCommitsWithBasehead` instead of `compareCommits` because the latter is deprecated: https://octokit.github.io/rest.js/v21/#repos-compare-commits

Reply:
The docs say "Both `:base` and `:head` must be branch names in `:repo`.", isn't that going to break with tag names?